app/helpers/field_validation_helper.rb

```diff
@@ -53,6 +53,19 @@ def validate_checkbox_field(page, values:, allowed_values:)
     []
   end
 
+  def validate_date_fields(year, month, day, field)
+    return [] if year.blank? && month.blank? && day.blank?
+
+    invalid_fields = []
+    invalid_fields << { field: field.to_s, text: t("coronavirus_form.errors.missing_year", field: field).humanize } if year.blank?
+    invalid_fields << { field: field.to_s, text: t("coronavirus_form.errors.missing_month", field: field).humanize } if month.blank?
+    invalid_fields << { field: field.to_s, text: t("coronavirus_form.errors.missing_day", field: field).humanize } if day.blank?
+    unless(invalid_fields != [] || Date.valid_date?(year.to_i, month.to_i, day.to_i))
```

Comment:
Do we have any steer on over 18 or under 18 validations yet?

Reply:
Good question, I've asked if we want to evaluate the date beyond checking it's valid.

Reply:
I'm told we shouldn't change behaviour based on the data here.